test/unit-tests/strategies/lyraDeploy.ts

```diff
@@ -0,0 +1,59 @@
+import { constants, evm, lyraMarkets, TestSystemContractsType, utils } from '@lyrafinance/core';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import chai, { expect } from 'chai';
+import { solidity } from 'ethereum-waffle';
+import { BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+chai.use(solidity);
+
+describe('Integration Test', () => {
+  let account: SignerWithAddress;
+  let testSystem: TestSystemContractsType;
+
+  let boardIds: BigNumber[];
+  let strikeIds: BigNumber[];
+
+  let snap: number;
+
+  before(async () => {
+    [account] = await ethers.getSigners();
+    testSystem = await lyraMarkets.deploy(account, true);
+    await lyraMarkets.seed(account, testSystem);
+  });
+
+  beforeEach(async () => {
+    snap = await evm.takeSnapshot();
+  });
+
+  afterEach(async () => {
+    await evm.restoreSnapshot(snap);
+  });
+
+  it('will pay out long calls', async () => {
+    boardIds = await testSystem.optionMarket.getLiveBoards();
+    strikeIds = await testSystem.optionMarket.getBoardStrikes(boardIds[0]);
+
```

Comment:
seems like strikeIds[0] is 1500?  (so the payout is 500 at eth = 2000)
If so better comment here or add a simple 
```
expect(strikeIds[0].eq(utils.toBN('500')))
```
To make it more readable